lib/StripeResource.js

```diff
@@ -239,30 +239,46 @@ StripeResource.prototype = {
 
   _request: function(method, path, data, auth, options, callback) {
     var self = this;
+    var requestProcessor;
     var requestData;
 
     if (self.requestDataProcessor) {
-      requestData = self.requestDataProcessor(method, data, options.headers);
+      requestProcessor = self.requestDataProcessor(method, data);
```

Comment:
Since this now gets a function, rather than just returning the data, it might be clearer if it were called something like `requestDataProcessorFn` or `getRequestDataProcessorFn` I think?

Reply:
The variable also might be a bit clearer if it were named `processRequestData` since that's what it's going to to.

Reply:
thanks for the feedback. I'll update the variable names to be more clear.